src/listeners/mentionPrefixOnly.ts

```diff
@@ -0,0 +1,21 @@
+import {Listener} from '@sapphire/framework';
+import {BediEmbed} from '../lib/BediEmbed';
+import {Message} from 'discord.js';
+import {fetchPrefix, surroundStringWithBackTick} from '../utils/discordUtil';
+
+export class UserEvent extends Listener<'mentionPrefixOnly'> {
+  public async run(message: Message) {
+    const prefix = await fetchPrefix(message);
```

Comment:
If this is only used inside the if branch then you should initialize it there too